packages/http-server/README.md

```diff
@@ -8,8 +8,8 @@ Usage:
 import { createServer } from '@stoplight/prism-http-server';
 
 const server = createServer({
-  path: './api.oas2.json',
-  {}
+  operations,
```

Comment:
```suggestion
  await getHttpOperationsFromResource('./api.oas2.json'),
```

Additionally, all should be wrapped into async func.

Reply:
I've changed; ultimately this is an example and I think people will understand what we're talking about (async await) — I hope at least.

Reply:
Some will, some won't. I believe it would be a couple less wtf's addressed to us if we do docs working out of a box.
Leaving to choice to you!